packages/shorebird_cli/lib/src/shorebird_build_mixin.dart

```diff
@@ -59,4 +59,31 @@ mixin ShorebirdBuildMixin on ShorebirdCommand {
       );
     }
   }
+
+  Future<void> buildApk() async {
+    const executable = 'flutter';
+    final arguments = [
+      'build',
+      // This is temporary because the Shorebird engine currently
```

Comment:
Is this true?

Reply:
good call we can probably remove this comment 😄